stat/int.go

```diff
@@ -0,0 +1,27 @@
+package stat // import "github.com/influxdata/influxdb/stat"
+
+import "sync"
+
+type Int struct {
+	mu    sync.RWMutex
```

Comment:
The [note](https://golang.org/pkg/sync/atomic/#pkg-note-BUG) in `sync/atomic` states that structs start off as 64-bit aligned. Assuming that a struct only contains 64-bit values, you shouldn't see alignment issues.